<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Login</class>
 <widget class="QWidget" name="Login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>360</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
*{
	font: normal 20px “微软雅黑”;
}
</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>-11</x>
     <y>0</y>
     <width>711</width>
     <height>360</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>“微软雅黑”</family>
     <pointsize>-1</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">#label{
 background-image: url(:/images/login.jpg); 

}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../../images/login.jpg</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>339</x>
     <y>119</y>
     <width>43</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>“微软雅黑”</family>
     <pointsize>-1</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">*{
color: rgb(255,255,255);
}</string>
   </property>
   <property name="text">
    <string> ID</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>339</x>
     <y>159</y>
     <width>43</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>“微软雅黑”</family>
     <pointsize>-1</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">*{
color: rgb(255,255,255);

}</string>
   </property>
   <property name="text">
    <string>密码:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_name">
   <property name="geometry">
    <rect>
     <x>389</x>
     <y>120</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>BY-101</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_pswd">
   <property name="geometry">
    <rect>
     <x>389</x>
     <y>169</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <property name="text">
    <string>123456</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>10</y>
     <width>371</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>“微软雅黑”</family>
     <pointsize>-1</pointsize>
     <weight>99</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">*{

	background-color:transparent;
	color: rgb(255,255, 255);
	font-size : 40px;
	font-weight:900;
	
}
</string>
   </property>
   <property name="text">
    <string>BOOK   STORE</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_login">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>220</y>
     <width>93</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{

    

    

}

QPushButton:hover{


	border:1px solid blue;
	background-color: rgb(0,118,143);

}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/images/loginbut.png</normaloff>:/images/loginbut.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_register">
   <property name="geometry">
    <rect>
     <x>359</x>
     <y>219</y>
     <width>93</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{

    

    

}

QPushButton:hover{


	border:1px solid blue;
	background-color: rgb(0,118,143);

}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/images/register.png</normaloff>:/images/register.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_close">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>0</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{

    

    

}

QPushButton:hover{


	border:1px solid blue;
	background-color: rgb(0,118,143);

}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/images/close2.png</normaloff>:/images/close2.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
</ui>
